.gitignore

```diff
@@ -1,3 +1,21 @@
+/modules/public/bindata.go
```

Comment:
Please revert these changes, there is so much junk inside that shouldn't be part of a project gitignore. Build your own global gitignore that ignores things like that.